infrastructure-petstore-rest-client/build.gradle

```diff
@@ -0,0 +1,9 @@
+dependencies {
+    api 'org.springframework.cloud:spring-cloud-starter-openfeign'
+    implementation 'org.springframework:spring-context'
```

Comment:
Isn't the spring-context always included in the spring boot starters?

Reply:
true.